repos/fdbt-netex-output/src/netex-convertor/netexGenerator.ts

```diff
@@ -352,6 +354,12 @@ const netexGenerator = async (ticket: Ticket, operatorData: Operator[]): Promise
     const updatePriceFareFrame = (priceFareFrame: NetexObject): NetexObject => {
         const priceFareFrameToUpdate = { ...priceFareFrame };
 
+        if (!('pricingByDistance' in ticket.products[0])) {
+            delete priceFareFrameToUpdate.geographicalUnits;
+            delete priceFareFrameToUpdate.tariffs.Tariff.GeographicalUnitRef;
+            delete priceFareFrameToUpdate.tariffs.Tariff.geographicalIntervals;
+        }
```

Comment:
else {

tariff.geographicalIntervals.GeographicalInterval = getGeographicalIntervals(
                    ticket.products[0] as PriceByDistanceProduct,
                );

}

Reply:
im not sure i understand what you mean here?